timer.py

```diff
@@ -0,0 +1,30 @@
+import time 
+
+def printTime(func):
+    def inner(*args, **kwargs):
+        print("Time of execution of decorated function is:", func(*args, **kwargs))
+    return inner
+
+def timer(func):
+    def inner(*args, **kwargs): 
+        timeStart = time.time()
+        func(*args, **kwargs)
+        timeStop = time.time()
```

Comment:
According to [PEP-8](https://peps.python.org/pep-0008/#descriptive-naming-styles) variable names should be lowercase with underscores, so `time_stop` in this case.